src/index.ts

```diff
@@ -374,7 +374,7 @@ const urlSafeKey = new entity.URLSafeKey();
  * });
  */
 class Datastore extends DatastoreRequest {
-  clients_: Map<string, {}>;
+  clients_: Map<string, {[k: string]: Function}>;
```

Comment:
You could use `import {ClientStub} from 'google-gax';` here

Reply:
Nice! Done
Thanks